### DISCUSSION TITLE
Requesting Icon for depicting duration of a call

### DISCUSSION THREAD

**Discussion by srgupta-twilio** [twilio-labs/paste#3035] [Request: Icon] [answered]
Hi, I'd need an icon for customer interaction history to demonstrate the duration of the call. 
I believe this fits the purpose - https://www.streamlinehq.com/icons/streamline-light?search=time
To be used here - 
<img width="596" alt="Screenshot 2023-02-13 at 11 35 41" src="https://user-images.githubusercontent.com/122378627/218447653-53b8ccbd-3572-40ad-8825-7a40c98f5f5d.png">

**Comment by TheSisb** [comment]
Hi @srgupta-twilio,

We'll be adding something for this in our next sprint. We'll report back when it's ready. Thanks!

**Comment by loreina** [comment]
Hi @srgupta-twilio, can you please [file a dependency](https://airtable.com/shrvMvL3BG6Zm0f9l) for the icon? The team is already working on this, but the form helps us keep track of our dependencies. We'll follow up once the icon is ready for use. 

Thank you!

**Comment by srgupta-twilio** [comment]
Thank you team :) Done!

**Comment by serifluous** [comment]
Hey @srgupta-twilio ! We added the Time icon to Figma in the "[NEW] Paste Components" library. We'll follow up again here when it's added to code

Let us know if you have any feedback on it.

**Comment by srgupta-twilio** [comment]
Thanks :)

**Comment by nkrantz** [comment, marked as answer]
Hey @srgupta-twilio, feel free to follow along with this PR (#3095). When it merges, all of the new icons will be available for use!

**Comment by srgupta-twilio** [comment]
Thank you!
